CoreTweet.Shared/Objects/DirectMessageEvents.cs

```diff
@@ -126,16 +126,22 @@ public class MessageAttachment : CoreBase
         public MediaEntity Media { get; set; }
     }
 
-    public class QuickReply : CoreBase
+    public abstract class QuickReply : CoreBase
     {
         [JsonProperty("type")]
-        public string Type { get; set; }
+        public virtual string Type { get; set; }
+    }
+
+    public class OptionsQuickReply : QuickReply
+    {
+        [JsonProperty("type")]
+        public override string Type { get; set; } = "options";
 
         [JsonProperty("options")]
-        public QuickReplyOption[] Options { get; set; }
+        public OptionsQuickReplyOption[] Options { get; set; }
     }
 
-    public class QuickReplyOption : CoreBase
+    public class OptionsQuickReplyOption : CoreBase
```

Comment:
I'm not sure why this can't be simply `QuickReplyOptions`. Is there any reason for this?